cancer_ai/validator/competition_handlers/melanoma_handler.py

```diff
@@ -20,6 +20,8 @@
 from cancer_ai.validator.models import WanDBLogModelBase
 from .base_handler import BaseCompetitionHandler, BaseModelEvaluationResult
 
+MAX_INVALID_ENTRIES = 2  # Maximum number of invalid entries allowed in the dataset
```

Comment:
why did you fix melanoma handler? did you test it?  afaik this code doesn't work at all